ui/src/views/mint/github-step/steps/github-repo-configuration/repo-configuration-body/repo-configuration-body.tsx

```diff
@@ -31,7 +38,7 @@ export const RepoConfigurationBody = () => {
         />
         <RepoBranchCommitFields />
         <Button
-          disabled={!branchName.value || !commitHash}
+          disabled={!isValid}

```

Comment:
I wonder if this is correct since this `isValid` state is related to the full form validation, maybe doublecheck this.

Reply:
it is


https://user-images.githubusercontent.com/52458215/228348930-0a5f4f6a-6e7a-4e95-b7ab-2cb4dd208df4.mov